stories/datafusion.stories.js

```diff
@@ -0,0 +1,110 @@
+import { renderTilesList, setAuthTokenWithOAuthCredentials } from './storiesUtils';
+
+import {
+  ProcessingDataFusionLayer,
+  CRS_EPSG3857,
+  BBox,
+  MimeTypes,
+  ApiType,
+  S2L1CLayer,
+  S2L2ALayer,
+} from '../dist/sentinelHub.esm';
+
+if (!process.env.INSTANCE_ID) {
+  throw new Error('INSTANCE_ID environment variable is not defined!');
+}
+
+if (!process.env.S2L1C_LAYER_ID) {
+  throw new Error('S2L1C_LAYER_ID environment variable is not defined!');
+}
+if (!process.env.S2L2A_LAYER_ID) {
+  throw new Error('S2L2A_LAYER_ID environment variable is not defined!');
+}
+
+const instanceId = process.env.INSTANCE_ID;
+const layerS2L1CId = process.env.S2L1C_LAYER_ID;
+const layerS2L2AId = process.env.S2L2A_LAYER_ID;
```

Comment:
If it helps, in `index.stories.js` these names are formulated in a different order (`s2l2aLayerId`)

Reply:
Ok, renamed.

Reply:
Ok, resolving.